package.json

```diff
@@ -37,6 +37,7 @@
   "devDependencies": {
     "@babel/plugin-transform-runtime": "7.12.1",
     "@size-limit/preset-big-lib": "4.9.0",
+    "@storybook/addon-controls": "^6.1.10",
```

Comment:
Please update to latest next-v3 and use the updated structure for storybook project. Keep related dependencies within `src/playground` folder. In the case of this addon, it will be `@ecl/twig-ec-storybook` and `@ecl/twig-eu-storybook` using the addon in `main.js`, thus keeping the dep as close as possible to its usage.

Reply:
Please keep versions of storybook packages in version sync. Other addons in the repo make use of `6.1.7`. As storybook team releases all addons from a mono-repo like ecl, it happens that assumptions of version sync can introduce bugs.